dartagnan/src/test/java/com/dat3m/dartagnan/spirv/basic/SpirvAssertionsTest.java

```diff
@@ -78,30 +77,23 @@ public static Iterable<Object[]> data() throws IOException {
                 {"builtin-all-321.spv.dis", 1, PASS},
                 {"branch-cond-ff.spv.dis", 1, PASS},
                 {"branch-cond-ff-inverted.spv.dis", 1, PASS},
-                {"branch-cond-bf.spv.dis", 1, FAIL},
+                {"branch-cond-bf.spv.dis", 1, UNKNOWN},
                 {"branch-cond-bf.spv.dis", 2, PASS},
-                {"branch-cond-bf.spv.dis", 3, PASS},
-                {"branch-cond-fb.spv.dis", 1, FAIL},
+                {"branch-cond-fb.spv.dis", 1, UNKNOWN},
                 {"branch-cond-fb.spv.dis", 2, PASS},
-                {"branch-cond-fb.spv.dis", 3, PASS},
                 {"branch-cond-struct.spv.dis", 1, PASS},
                 {"branch-cond-struct-read-write.spv.dis", 1, PASS},
                 {"branch-race.spv.dis", 1, PASS},
-                {"branch-loop.spv.dis", 2, FAIL},
+                {"branch-loop.spv.dis", 2, UNKNOWN},
                 {"branch-loop.spv.dis", 3, PASS},
-                {"branch-loop.spv.dis", 4, PASS},
-                {"loop-struct-cond.spv.dis", 1, FAIL},
+                {"loop-struct-cond.spv.dis", 1, UNKNOWN},
                 {"loop-struct-cond.spv.dis", 2, PASS},
-                {"loop-struct-cond.spv.dis", 3, PASS},
-                {"loop-struct-cond-suffix.spv.dis", 1, FAIL},
+                {"loop-struct-cond-suffix.spv.dis", 1, UNKNOWN},
                 {"loop-struct-cond-suffix.spv.dis", 2, PASS},
-                {"loop-struct-cond-suffix.spv.dis", 3, PASS},
-                {"loop-struct-cond-sequence.spv.dis", 2, FAIL},
+                {"loop-struct-cond-sequence.spv.dis", 2, UNKNOWN},
                 {"loop-struct-cond-sequence.spv.dis", 3, PASS},
-                {"loop-struct-cond-sequence.spv.dis", 4, PASS},
-                {"loop-struct-cond-nested.spv.dis", 2, FAIL},
+                {"loop-struct-cond-nested.spv.dis", 2, UNKNOWN},
                 {"loop-struct-cond-nested.spv.dis", 3, PASS},
-                {"loop-struct-cond-nested.spv.dis", 4, PASS},
```

Comment:
- Why are you removing some tests?

- Was it the case that we were giving wrong results before (due to not really evaluating the condition after threads terminated), thus some previous FAIL are now UNKNOWN?

Reply:
It seems the Natalia removed tests with unnecessarily high bounds, i.e., we don't need to use B=4 if B=3 already gives PASS.